src/core/views.py

```diff
@@ -4195,7 +4202,9 @@ def create_completed_proposal_review_form(proposal, review_id):
         data = json.loads(review_assignment.results.data)
 
         for relation in relations:
-            v = data[relation.element.name]
+            v = data.get(
+                relation.element.name.encode('ascii', 'xmlcharrefreplace')
```

Comment:
You're doing this over and over throughout this process. Wouldn't it be useful to create a single method that does this, limiting the possibility of getting `.encode('ascii', 'xmlcharrefreplace')` wrong with any of the points it's used?